src/app/public/modules/phone-field/phone-field-input.directive.ts

```diff
@@ -61,6 +61,19 @@ const SKY_PHONE_FIELD_VALIDATOR = {
 };
 // tslint:enable
 
+/**
+ * Creates a button, search input, and text input for entering and validating
+ * international phone numbers. Place this attribute on an `input` element, and wrap
+ * that element in a `sky-phone-field` component. By default, the country selector
+ * button displays a flag icon for the default country, and the phone number input
+ * displays a sample of the correct phone number format. When users select the country
+ * selector button, they expose the country search input, which is
+ * [an autocomplete input](https://developer.blackbaud.com/skyux/components/autocomplete)
+ * that allows them to select different countries. When users enter `+` followed by an
+ * international dial code in the phone number input, the country automatically switches
+ * to the country associated with the dial code.
+ * @required
```

Comment:
How do we make the directive description a line in the table like in character count at https://developer.blackbaud.com/skyux/components/character-count#charactercountinputdirectiveproperties? I assume the directive description should be displayed one way or the other for consistency. Is the character count right, or is this? @Blackbaud-SteveBrush

Reply:
Nevermind. Just realized that character count docs aren't in phase 2. I assumed they were because there are JSDoc comments in place.